src/components/CommentPost.jsx

```diff
@@ -0,0 +1,22 @@
+/* eslint-disable jsx-a11y/label-has-associated-control */
+
+// !!! REMOVE THE ABOVE COMMENT IF YOU PLAN TO CHANGE THIS FILE !!!
+
+export default function CommentPost(){
+
+
+    return(
+        <form className="mb-6">
+        <div className="py-2 px-4 mb-4 bg-white rounded-lg rounded-t-lg border border-gray-200 dark:bg-gray-800 dark:border-gray-700">
+            <label htmlFor="comment" className="sr-only"> Your comment </label>
+            <textarea id="comment" rows="6" 
+            className="px-0 w-full text-sm text-gray-900 border-0 focus:ring-0 focus:outline-none dark:text-white dark:placeholder-gray-400 dark:bg-gray-800"
+            placeholder="Write a comment..." required/> 
```

Comment:
@NijyarGS to fix eslint error, you should put the textarea into label , I faced that before and fixed it sth like this:

```suggestion
            <label htmlFor="comment" className="sr-only"> Your comment  
            <textarea id="comment" rows="6" 
            className="px-0 w-full text-sm text-gray-900 border-0 focus:ring-0 focus:outline-none dark:text-white dark:placeholder-gray-400 dark:bg-gray-800"
            placeholder="Write a comment..." required/> 
            </label>
```